platform_versioned_docs/version-23.2.0/enterprise/advanced-topics/manual-aws-batch-setup.mdx

```diff
@@ -158,25 +160,23 @@ Create a launch template to configure the EC2 instances deployed by Batch jobs:
 
     #cloud-config
     write_files:
-      - path: /root/tower-forge.sh
+      - path: /root/seqera-ce.sh
         permissions: 0744
         owner: root
         content: |
           #!/usr/bin/env bash
-          exec > >(tee /var/log/tower-forge.log|logger -t TowerForge -s 2>/dev/console) 2>&1
-          ##
           yum install -q -y jq sed wget unzip nvme-cli lvm2
-          wget -q https://s3.amazonaws.com/amazoncloudwatch-agent/amazon_linux/amd64/latest/amazon-cloudwatch-agent.rpm
+          wget -q https://amazoncloudwatch-agent.s3.amazonaws.com/amazon_linux/amd64/latest/amazon-cloudwatch-agent.rpm
           rpm -U ./amazon-cloudwatch-agent.rpm
           rm -f ./amazon-cloudwatch-agent.rpm
-          curl -s https://nf-xpack.seqera.io/amazon-cloudwatch-agent/config-v0.3.json \
-            | sed 's/$FORGE_ID/<your prefix>/g' \
+          curl -s https://nf-xpack.seqera.io/amazon-cloudwatch-agent/custom-v0.1.json \
+          #  | sed 's/custom-id/<your custom ID>/g' \
```

Comment:
If commented, worth to remove it

Reply:
If it's removed completely, users don't have the option to add their own custom prefix to log streams. There's an instruction below the LT telling them to uncomment to customize, or omit to keep default `custom-id`. Better to give users the option and avoid them having to ask for it / try insert the line themselves and potentially break the LT.